### PR TITLE
19 js calendar

### LINKED ISSUE #19: jQuery calendar
Create partial calendar form that can store dates in params

Verify that it can be loaded into another view

### PR DESCRIPTION
**Pull request by Sh1pley**
screen shot 2017-02-08 at 8 21 34 am
screen shot 2017-02-08 at 8 21 22 am
screen shot 2017-02-08 at 8 21 40 am

Closes #19

@njgheorghita @riverswb @ski-climb @tmikeschu 

adds jquery pop up calendar to check in check out fields

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ gem 'coffee-rails', '~> 4.2'
 
 # Use jquery as the JavaScript library
 gem 'jquery-rails'
+gem 'jquery-ui-rails'
```
Comment (tmikeschu) [inline Gemfile:29]: You and I.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -13,4 +13,5 @@
 //= require jquery
 //= require bootstrap-sprockets
 //= require jquery_ujs
+//= require jquery-ui
```
Comment (tmikeschu) [inline app/assets/javascripts/application.js:16]: Someone read the docs! Nice.

File: app/assets/javascripts/dateselect.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+$(document).on('ready page:load', function() {
+  $('#Check_In').datepicker();
+});
+
+$(document).on('ready page:load', function() {
```
Comment (tmikeschu) [inline app/assets/javascripts/dateselect.js:5]: This is awesome!! Very readable code too.

File: app/views/home/show.html.haml [hunk 4]
```diff
@@ -5,9 +5,9 @@
     = form_tag(controller: :home, action: :show) do
 
       = text_field_tag "Destination", nil, placeholder: "Destination"
-      %section.dates
-        = date_field_tag "Check In"
-        = date_field_tag "Check Out"
+      %section.dates#datepicker
+        = text_field_tag "Check In", placeholder="Check In"
```
Comment (tmikeschu) [inline app/views/home/show.html.haml:9]: Awesome! @ski-climb and I were struggling with the placeholder for a date field.

**Review by tmikeschu — COMMENTED** [review]
Nice work, @Sh1pley ! Can you throw up a screen shot?

**Comment by tmikeschu** [comment]
@njgheorghita @riverswb @ski-climb count me in for a merge endorsement.

**Review by riverswb — APPROVED** [review]

**Comment by Sh1pley** [comment]
@tmikeschu @riverswb @njgheorghita @ski-climb 

re opened because of my fails

**Comment by ski-climb** [comment]
Nice job @Sh1pley!  That was a surprisingly small amount of code to get that implemented.  Sweet job on the jQuery too.

**Pull request merged**